packages/api/src/resolvers/article/index.ts

```diff
@@ -643,10 +649,16 @@ export const setBookmarkArticleResolver = authorized<
         return { errorCodes: [SetBookmarkArticleErrorCode.NotFound] }
       }
 
-      // delete the page
+      // delete the page and its metadata
       const deleted = await updatePage(
         pageRemoved.id,
-        { state: ArticleSavingRequestStatus.Deleted },
+        {
```

Comment:
Do we also clear content and originalHTML when we delete?

Reply:
We do not clear them. hmm, my concern is we could use them for some analytics and stuff @jacksonh

Reply:
I'm open to clear them too since we are not using them now